content/millennium/dstu2/medications/medication-administration.md

```diff
@@ -0,0 +1,135 @@
+* Details of medication administered:
+  * [Dosage](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.dosage){:target="_blank"}
+  * [Route](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.dosage.route){:target="_blank"}
+  * [Quantity](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.dosage.quantity){:target="_blank"}
+  * [Site](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.dosage.site_x_){:target="_blank"}
+  * [Rate](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.dosage.rate_x_){:target="_blank"}
+
+
+<!-- ## Terminology Bindings
+
+<%= terminology_table(:medication_statement, :dstu2) %> -->
+
+<!-- ### Contained Medication Bindings
+
+<%= terminology_table(:contained_medication, :dstu2) %> -->
+
+## Search
+
+Search for MedicationAdministrations that meet supplied query parameters:
+
+    GET /MedicationAdministration?:parameters
+
+_Implementation Notes_
+
+* [MedicationAdministration.medication] may be a reference to a [contained] Medication when the Medication cannot be represented by a CodeableConcept because it contains a unique combination of ingredients. Medications in the system always exist within the context of a MedicationAdministration and cannot be be referenced independently.
+
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: true, system: true) %>
+
+
+### Parameters
+
+
+ Name           | Required?          | Type          | Description
+----------------|--------------------|---------------|-----------------------------------------------------------------------------------------------
+`_id`           | This, or `patient` | [`token`]     | The logical resource id associated with the resource.
+`patient`       | This, or `_id`     | [`reference`] | The identifier of the patient who has received the medication administration. Example: `patient=12345`
+`status`        | N                  | [`token`]     | The status of the medication administration, may be a list separated by commas.  Example: `status=active`
+`practitioner`  | N                  | [`reference`] | The identifier of the  performing clinician, may be a list separated by commas. Example: `practitioner=12345`
+`notgiven`      | N                  | [`token`]     | Administrations that were not made. A value of 'true' will search for these, and a value of 'false' will exclude them. Example: `true`
+`effectivetime` | N                  | [`date`]      | The date/time when the Procedure was performed. Must use both ge and le prefixes in order to search. Example: `effectivetime=ge2010-08-18&effectivetime=le2015-07-17`
+[`_count`]      | N                  | [`number`]    | The maximum number of results to return. Defaults to `_count=50`.
```

Comment:
Remove the default. This can change based on changes in the system for performance adjustment

Reply:
Removed: https://github.com/cerner/fhir.cerner.com/pull/169/commits/037712040917c1d556c6c8c76363db6eb41412c2#diff-39322533700e1bb799f2f6d703a9ae74L66